src/Cello/error_Error.hpp

```diff
@@ -245,6 +245,8 @@
   {  if (!(A)) { m2_(stderr,"ERROR",__FILE__,__LINE__,F,M,A1,A2,A3,A4,A5,A6); t_(); } }
 #define ASSERT7(F,M,A1,A2,A3,A4,A5,A6,A7,A)				\
   {  if (!(A)) { m2_(stderr,"ERROR",__FILE__,__LINE__,F,M,A1,A2,A3,A4,A5,A6,A7); t_(); } }
+#define ASSERT8(F,M,A1,A2,A3,A4,A5,A6,A7,A8,A)				\
+  {  if (!(A)) { m2_(stderr,"ERROR",__FILE__,__LINE__,F,M,A1,A2,A3,A4,A5,A6,A7,A8); t_(); } }
```

Comment:
Would be great if the arguments are be protected by `()`, but likely sth for a later PR.